styles/_search_results.scss

```diff
@@ -124,27 +147,16 @@
   }
 }
 
-/* Non-latin search results highlight */
+/* By default use bold instead of italic to highlight */
+.search-results-description em {
+  font-style: normal;
+  font-weight: bold;
+}
 
 /* Add a yellow background for Chinese */
 html[lang|="zh"] {
-  .search-result-description em {
```

Comment:
I can't find references to `search-result-description` so I assume it was wrong and we're just correcting it to `search-results-description` (?)

Reply:
yes. this should have been a typo. now correct it.